scripts/exglobal_archive.py

```diff
@@ -31,16 +31,20 @@ def main():
             'restart_interval_gdas', 'restart_interval_gfs',
             'AERO_ANL_RUN', 'AERO_FCST_RUN', 'DOIBP_WAV', 'DO_JEDIOCNVAR',
             'NMEM_ENS', 'DO_JEDIATMVAR', 'DO_VRFY_OCEANDA', 'FHMAX_FITS',
-            'IAUFHRS', 'DO_FIT2OBS']
+            'IAUFHRS', 'DO_FIT2OBS', 'NET']
```

Comment:
This also needs to be appended to exgdas_enkf_earc.py here:
https://github.com/NOAA-EMC/global-workflow/blob/2e4f4b7671cfec83331ec26de39218543d7b9a6d/scripts/exgdas_enkf_earc.py#L31